src/settings-tab.ts

```diff
@@ -202,6 +218,7 @@ export class MindMapSettingsTab extends PluginSettingTab {
           })
       );
 
+    const color_default =
     new Setting(containerEl)
```

Comment:
- move this new Setting to the previous line, right after the `=`, followed by a space char.